### DISCUSSION TITLE
Rip HDDVDs with ARM

### DISCUSSION THREAD

**Discussion by mdalen1** [automatic-ripping-machine/automatic-ripping-machine#961] [Q&A] [answered]
I have recently purchased an Xbox HDDVD player to rip some HDDVDs I was given. The player connects by USB to the ripper, and works fine when ripping DVDs. However HDDVDs exit with the message "Could not identify disc. Exiting."

I saw from the Issue tracker that people have ripped HDDVDs successfully earlier. Any tips on getting this to work?

**Comment by shitwolfymakes** [comment]
HDDVD support does not exist in ARM currently, and adding it would depend on whether or not MakeMKV supports it

**Reply by mdalen1** [reply]
Thanks for the quick reply. I have tested the same drive and discs with MakeMKV on Windows 10, and HDDVDs are detected and ripped by MakeMKV. I am not sure if this means they are supported by MakeMKV for Linux.

**Reply by shitwolfymakes** [reply, marked as answer]
Fascinating, good to know. That being said, adding a ripper mode for HDDVDs will take a lot of work, so expect to need to rip them manually for the foreseeable future

**Reply by mdalen1** [reply]
Ok, thanks, I'll rip them on Windows then. If it is ever decided to add support in the future let me know if I can help with any testing.
